api/LICENSE-BSD-3-Clause

```diff
@@ -0,0 +1 @@
+../LICENSE-BSD-3-Clause
```

Comment:
Is the license file required for each crate?

Reply:
It's